README.md

```diff
@@ -1415,15 +1448,81 @@ negotiation.
 - Finally, consider the [common scenarios](#common-scenarios) for exercising options, discussed
   below.
 
+### Evaluating startup offers
+
+Startups present their own set of risks, since many startups are risky and unproven
+businesses.
+
+🔹 If you’re considering working for a startup, there are a lot of questions to ask in
+order to assess the state of the company’s business.
+Startups are understandably careful about sharing financial information, so you may not
+get full answers to all of these, but you should at least ask:
+
+- *How much money has the company raised (including in how many rounds, and when)?*
+- *What did the last round value the company at?*
+- *Will the company likely raise more capital soon?*
+- *How long will the company’s current funding last?*
+  (This will likely be given at the current burn rate, or how quickly a company is spending
+  its funding, so will likely not include calculations for things like future employee
+  salaries.)
+- *What is the hiring plan?*
+  (How many people over what time frame?)
+- *What is the revenue now, if any?
+  What are the revenue goals/projections?*
+- *Where do you see this company in 1 year and 5 years, in terms of revenue, number of
+  employees, and market position?*
+- 🚧 Combine with and link to
+  [Wealthfront’s crucial questions](https://blog.wealthfront.com/stock-options-14-crucial-questions/)
+
+### Some stock option scenarios
```

Comment:
Not sure about the logic of this between "Evaluating startup offers" and "Offers and Negotiations" (or under the Evaluating Equity Compensation heading at all). Logic, philosophy, and methods of exercise are likely to be a highly searched component of this topic. Consider giving these "scenarios" (choices!) their own section where we can add some depth on each.

Reply:
I've renamed the section "Evaluating startups" since that is what it really is.